### PR TITLE
Hitsounds for CoC, rapiers. Throwsound for bolas.

### PR DESCRIPTION
**Pull request by Cheridan**
slash.ogg is a fucking travesty

### CODE REVIEW

**Comment by GunHog** [comment]
Yay, more sounds!! Without a sound guy, things have been very limited! (Cuboos seems to have vanished)

**Comment by PKPenguin321** [comment]
links for easy listening:
[rapierhit.ogg](http://puu.sh/qufdW/a7e6b3ea05.mp3)
[chainhit.ogg](http://puu.sh/quffj/1afd51bf84.mp3)
[bolathrow.ogg](http://puu.sh/qufhg/a49f9681f6.mp3)

- you've got about .2 seconds of empty noise at the start of rapierhit.ogg
- chain sound is imo oddly low quality compared to other sounds but still sounds mostly okay
- bola throw sound is fantastic

**Comment by PKPenguin321** [comment]
also: "Sounds PR Without Linking Sounds" tag when

**Comment by Cheridan** [comment]
just go to the files changed tab its seriously not hard

**Comment by PKPenguin321** [comment]
yeah but like for some reason whenever i try to listen to a sound hosted by github the player is like this and doesnt work
aaaaa

**Comment by GunHog** [comment]
Same issue for me.

**Comment by Kromgar** [comment]
>Rapier hits sound like limbs being severed. Shouldnt even be a rapier

File: code/game/objects/items/weapons/handcuffs.dm [hunk 1]
```diff
@@ -320,6 +320,11 @@
 	origin_tech = "engineering=3;combat=1"
 	var/weaken = 0
 
+/obj/item/weapon/restraints/legcuffs/bola/throw_at(atom/target, range, speed, mob/thrower, spin)
+	if(!..())
+		return
+	playsound(src.loc,'sound/weapons/bolathrow.ogg', 75, 1)
```
Comment (Fox-McCloud) [inline code/game/objects/items/weapons/handcuffs.dm]: Don't use this, use the ultra-rare `throwhitsound` var.

Comment (PKPenguin321) [inline code/game/objects/items/weapons/handcuffs.dm]: doesnt that play on hit though? the sound is like a whoosh of it flying through the air, not an impact sound

Comment (Fox-McCloud) [inline code/game/objects/items/weapons/handcuffs.dm]: Ehhhhhhhhh, I suppoooooseee!

**Comment by Jordie0608** [comment]
Whats the source for these sounds?

**Pull request merged**